subworkflows/local/arg.nf

```diff
@@ -49,14 +55,16 @@ workflow ARG {
         ch_versions = ch_versions.mix( AMRFINDERPLUS_RUN.out.versions )
 
     // Reporting
-        HAMRONIZATION_AMRFINDERPLUS ( AMRFINDERPLUS_RUN.out.report, 'json', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
-        ch_versions = ch_versions.mix( HAMRONIZATION_AMRFINDERPLUS.out.versions )
-        ch_input_to_hamronization_summarize = ch_input_to_hamronization_summarize.mix( HAMRONIZATION_AMRFINDERPLUS.out.json )
+        HAMRONIZATION_AMRFINDERPLUS_TSV ( AMRFINDERPLUS_RUN.out.report, 'tsv', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
+        HAMRONIZATION_AMRFINDERPLUS_JSON ( AMRFINDERPLUS_RUN.out.report, 'json', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
+        ch_versions = ch_versions.mix( HAMRONIZATION_AMRFINDERPLUS_JSON.out.versions )
```

Comment:
Every named process should be mixed, even if the same module itself. But see Jasmin's comment below.